src/packages/@ncigdc/components/SurvivalPlotWrapper.js

```diff
@@ -211,7 +211,6 @@ const SurvivalPlotWrapper = ({
                 ))}
             </Row>
           </div>
-
```

Comment:
Why is this change necessary?

I was trying to fix it using github, but it messes the spacing.
Mind undoing it?

Reply:
merge funkiness probably. hang on